### PR TITLE
added Intellivoid SpamProtection

### PR DESCRIPTION
**Pull request by Krishna-Singhal**

### CODE REVIEW

File: userge/plugins/admin/gban.py [hunk 1]
```diff
@@ -294,7 +294,30 @@ async def gban_at_entry(message: Message):
             async with aiohttp.ClientSession() as ses:
                 async with ses.get(f'https://api.cas.chat/check?user_id={user_id}') as resp:
                     res = json.loads(await resp.text())
```
Comment (rking32) [inline userge/plugins/admin/gban.py:296]: first handle this 
```python
if res['ok']:
    ...
else:
    # check intellivoid ...

**Comment by rking32** [comment]
y two requests ?
do your second request inside `else:` part.

cuz you know, second request is useless if 1st `res['ok'] = True`

**Comment by Krishna-Singhal** [comment]
> y two requests ?
> do your second request inside `else:` part.
> 
> cuz you know, second request is useless if 1st `res['ok'] = True`

yes 🤗

**Comment by rking32** [comment]
good

**Pull request merged**